docs/_Research-Data-Management/03-md.md

```diff
@@ -10,12 +10,11 @@ hide: false
 
 # Metadata
 
-<img src="https://upload.wikimedia.org/wikipedia/commons/e/ee/Metadata_is_a_love_note_to_the_future_%288071729256%29.jpg" alt="Metadata Is a Love Note to the Future - Jason Scott" width="200"/>
+<img src="https://upload.wikimedia.org/wikipedia/commons/e/ee/Metadata_is_a_love_note_to_the_future_%288071729256%29.jpg" alt="Metadata Is a Love Note to the Future - Jason Scott" width="200"/> <br>
```

Comment:
The whole purpose of Markdown is to reduce the tag-heavy syntax, therefore we should refrain from using HTML tags for clarity **especially** when there are perfectly fine way to do it in Markdown. See the specifications for Github Flavoured Markdown <https://github.github.com/gfm>

Especially images (see example 581) and line-break (see example 659) in relation to this line.

Now I guess that here the `img` tag was used to make use of the "width" attribute to control the size, which you cannot do with Github Flavoured Markdown. One workaround is to edit the image, or at least get it in a size closer to what we mean. Fortunately, this page uses images from wikimedia where different sizes are available directly. Here is a suggestion

```suggestion
![Metadata Is a Love Note to the Future - Jason Scott](https://upload.wikimedia.org/wikipedia/commons/thumb/e/ee/Metadata_is_a_love_note_to_the_future_%288071729256%29.jpg/179px-Metadata_is_a_love_note_to_the_future_%288071729256%29.jpg)  
```

Reply:
Thanks for the changes! I'll approve but have a question:
Regarding image, I checked against the ARUNA image on the knowledge base and they use 

> ![Aruna Object Storage Logo]({{ '/assets/img/aruna_dark_font.png' | relative_url }} "Aruna Object Storage Logo"){:width="20%"} 

is this possible for absolute hyperlinks as well?